extensions/amp-a4a/0.1/test/test-amp-a4a.js

```diff
@@ -201,7 +208,15 @@ describe('amp-a4a', () => {
     const child = element.querySelector('iframe[src]');
     expect(child).to.be.ok;
     expect(child.src).to.have.string(srcUrl);
-    expect(child.getAttribute('name')).not.to.be.ok;
+    const nameData = child.getAttribute('name');
+    expect(nameData).to.be.ok;
+    expect(JSON.parse.bind(null, nameData), nameData).not.to.throw(Error);
+    const attributes = JSON.parse(nameData);
+    expect(attributes).to.be.ok;
+    expect(attributes._context).to.be.ok;
+    if (!attributes._context.amp3pSentinel) {
```

Comment:
Ah.  You're doing essentially the same test here as above.  I suggest factoring the `nameData` validation out to a common function that you call from both places.